delfin/tests/unit/api/fakes.py

```diff
@@ -439,7 +439,7 @@ def fake_get_capabilities(context, storage_id):
                         "description": "Average time taken for an IO "
                                        "operation in ms"
                     },
-                    "requests": {
+                    "iops": {
                         "unit": "IOPS",
                         "description": "Input/output operations per second"
```

Comment:
will the description change based upon https://github.com/sodafoundation/architecture-analysis/pull/115/files

Reply:
Driver capability can override the description based on the calculation they make.. schema will just check a description is provided .